docs/Release_notes.md

```diff
@@ -1,6 +1,39 @@
 JPO-ODE Release Notes
 ----------------------------
 
+Version 4.1.0, released January 2025
+----------------------------------------
+### **Summary**
+This release finishes up the Spring Kafka migration with several classes and data flow diagram updates. Unused code has been removed to reduce 
+maintenance overhead, and continuous integration has been configured via Azure Pipelines. Docker Compose has been updated with all required 
```

Comment:
suggestion: the azure pipelines work is just on the CDOT side of things right? Should we leave that portion out since it doesn't apply to the overall product?

Reply:
Same with the Azure Artifacts reference below, the deployments run through separate process so I think we can leave those details out

Reply:
Removed Azure pipeline / artifact references from the release notes!